config_manager.py

```diff
@@ -20,21 +20,34 @@ def load_config(path, name):
 
 def manage_version(path, name):
     from glob import glob
-    configs = sorted(glob(os.path.splitext(os.path.join(path, name))[0] + '*'))
+    configs = sorted(glob('_'.join(os.path.splitext(os.path.join(path, name))[0].split('_')[:-1]) + '*'))
```

Comment:
why do you need to use os.path.join(path, name) if you would use os.path.splitext?
how about

        configs = sorted(glob(path[:path.rfind('_')] + '*')))

or if you really need os.path.join(path, name)

        root, _ = os.path.splitext(os.path.join(path, name))
        configs = sorted(glob(root[:root.rfind('_')] + '*'))

Reply:
ex) path == './config', name == 'test.json', so the first suggested method can't be same result,
and your second method is faster than the original method, thank you :)